utils/swift_build_support/swift_build_support/products/swiftdriver.py

```diff
@@ -137,6 +136,16 @@ def run_build_script_helper(action, host_target, product, args):
         helper_cmd += ['--cross-compile-hosts']
         for cross_compile_host in args.cross_compile_hosts:
             helper_cmd += [cross_compile_host]
+        build_toolchain_path = product.install_toolchain_path(host_target,
+                                                              force_local=True)
+        resource_dir = '%s/lib/swift' % build_toolchain_path
+        helper_cmd += [
+            '--cross-compile-config',
+            targets.StdlibDeploymentTarget.get_target_for_name(host_target).platform
+            .swiftpm_config(args, output_dir=build_toolchain_path,
+                            swift_toolchain=toolchain_path,
+                            resource_path=resource_dir),
+            '--prefix', install_destdir + args.install_prefix]
```

Comment:
This script never uses `install_destdir` or this `--prefix` flag, but just assumes `swift-driver` will install into the build toolchain's path. @artemcm, I can pull adding this flag out to native compilation too if you'd like.

Reply:
Yes, having a `--prefix` instead of always assuming the build toolchain as the destination would be great for non-cross-compilation flow as well!

Reply:
Will do as soon as this CI run finishes, as that will give some valuable info first.

Reply:
Added in latest commit.